components/automate-gateway/api/iam/v2beta/rules.proto

```diff
@@ -22,4 +22,20 @@ service Rules {
     option (chef.automate.api.iam.policy).resource = "iam:rules"; // TODO: not really iam, is it?
     option (chef.automate.api.iam.policy).action = "iam:rules:create";
   };
+
+  rpc GetRule (GetRuleReq) returns (GetRuleResp) {
+    option (google.api.http).get= "/iam/v2beta/rules/{id}";
+    option (chef.automate.api.policy).resource = "auth:rules:{id}";
+    option (chef.automate.api.policy).action = "get";
+    option (chef.automate.api.iam.policy).resource = "iam:rules:{id}";
+    option (chef.automate.api.iam.policy).action = "iam:rules:get";
+  };
+
+  rpc ListRules (ListRulesReq) returns (ListRulesResp) {
+    option (google.api.http).get= "/iam/v2beta/rules";
+    option (chef.automate.api.policy).resource = "auth:rules";
+    option (chef.automate.api.policy).action = "get";
```

Comment:
We are inconsistent with how we use `read` vs `get` in V1, but I don't see any `get` for list endpoints. Should probably be `read` but it also probably doesn't matter since no one will be using this in V1.